cocos/spine/skeleton.ts

```diff
@@ -749,25 +791,12 @@ export class Skeleton extends UIRenderer {
         return draw;
     }
 
-    @override
-    @type(Material)
-    @displayOrder(0)
-    @displayName('CustomMaterial')
-    get customMaterial () {
-        return this._customMaterial;
-    }
-    set customMaterial (val) {
-        this._customMaterial = val;
-        this.updateMaterial();
-        this.markForUpdateRenderData();
-    }
-
     protected _updateBuiltinMaterial (): Material {
         const material = builtinResMgr.get<Material>('default-spine-material');
         return material;
     }
 
-    public updateMaterial () {
+    protected updateMaterial () {
```

Comment:
updateMaterial should be public, but mark as `@engineInternal`, this is inherited API, shouldn't change its visibility